### PR TITLE
Update Ubuntu 1404 info, fixes #83

### LINKED ISSUE #83: Update instructions for restarting php-fpm (remove 14.04, add 18.04)
**Version of the documentation**
2.5

**Link to Redmine ticket (if applicable)**
12609 (release process)

**Pages (and section, if applicable) where the issue occurs**
https://www.accesstomemory.org/en/docs/2.5/admin-manual/maintenance/troubleshooting/#troubleshooting-restart-php-fpm

https://www.accesstomemory.org/en/docs/2.5/admin-manual/installation/upgrading/ 

https://www.accesstomemory.org/en/docs/2.5/admin-manual/installation/execution-limits/ 

https://www.accesstomemory.org/en/docs/2.5/admin-manual/installation/linux/ubuntu-trusty/ -  remove this page from the 2.5 documentation 

https://www.accesstomemory.org/en/docs/2.5/admin-manual/ (auto)

https://www.accesstomemory.org/en/docs/2.5/contents/ (auto)

**Description of the issue**
remove references to restarting php-fpm for Ubuntu 14.04 and add instructions for 18.04

### PR DESCRIPTION
**Pull request by corinnemrogers**

### CODE REVIEW

File: admin-manual/installation/execution-limits.rst [hunk 1]
```diff
@@ -167,29 +167,23 @@ If you do make changes to the PHP configuration poool and/or the ``php.ini``
...
+**In Ubuntu 16.04 with PHP 7:**
 
 .. code-block:: bash
 
-   sudo service php5-fpm restart
+   sudo systemctl restart php7.0-fpm
    php symfony cc
 
-**In Ubuntu 16.04 with PHP 7:**
+**In Ubuntu 18.04 with PHP 7:**
```
Comment (fiver-watson) [inline admin-manual/installation/execution-limits.rst]: I think we should specify 7.0 and 7.2 in these commands. It is technically possible for someone to have 16.04 with PHP 7.2, and vice versa.

File: admin-manual/installation/upgrading.rst [hunk 2]
```diff
@@ -250,42 +250,29 @@ and the atom-workers managed by the job scheduler.
...
-Ubuntu 14.04:
+Ubuntu 16.04:
 
 .. code-block:: bash
 
-   sudo service php5-fpm restart
+   sudo systemctl restart php7.0-fpm
 
-Ubuntu 16.04:
+Ubuntu 18.04:
```
Comment (fiver-watson) [inline admin-manual/installation/upgrading.rst]: Same here; let's specify 7.0 and 7.2

File: admin-manual/maintenance/troubleshooting.rst [hunk 3]
```diff
@@ -1519,62 +1501,61 @@ Which should return something like the following:
...
 ensure that the ``read_only`` setting is set to ``false``. See:
 
-* :ref:`config-app-yml` 
+* :ref:`config-app-yml`
 
 .. IMPORTANT::
 
-   There is also an environment variable maintained in ``/etc/php5/fpm/pool.d/atom.conf`` 
-   that can override the one in ``config/app.yml``. We recommend checking both 
+   There is also an environment variable maintained in ``/etc/php5/fpm/pool.d/atom.conf``
```
Comment (fiver-watson) [inline admin-manual/maintenance/troubleshooting.rst]: We should also update this path - see my general comment for the correct path in 7.2

**Review by fiver-watson — CHANGES_REQUESTED** [review]
Still finding some more places that should be reviewed and updated. Some examples:

* https://www.accesstomemory.org/en/docs/2.5/admin-manual/security/application/

The paths in my 18.04 Vagrant box for PHP 7.2 are:

* /etc/php/7.2/fpm/php-fpm.conf
* /etc/php/7.2/fpm/php.ini

We could check the current Vagrant box to confirm that 7.0 is similar. 

-----

* https://www.accesstomemory.org/docs/2.5/admin-manual/maintenance/debug/#using-qubit-dev-php

Right at the end of the section it mentions restarting PHP5-FPM, which should just be PHP-FPM, with both 7.0 and 7.2 versions given

-----

* https://www.accesstomemory.org/docs/2.5/dev-manual/api/api-intro/#enable-the-api-plugin

In the TIP admonition

------

* https://www.accesstomemory.org/docs/2.5/user-manual/import-export/oai-pmh/#enable-the-oai-plugin-in-atom

Also in the TIP admonition

-----

* https://www.accesstomemory.org/docs/2.5/admin-manual/maintenance/cli-tools/#purging-all-data

In the IMPORTANT admonition

-----

That's all the additional ones I could find!

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 4]
```diff
@@ -60,11 +66,16 @@ Ubuntu doesn't provide a package but you can download it directly from the
...
+Make sure that `Java <https://www.java.com/en/>`__ is installed. In this
 example we are going to use OpenJDK but Oracle's JVM would also work.
 
+   code content
+
 .. code-block:: bash
 
+   sudo add-apt-repository ppa:openjdk-r/ppa
+   sudo apt-get update
+
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: It's hard to tell in the diff, but it looks like there might be an extra blank linebreak between the apt-get update and the apt install openjdk command?

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 5]
```diff
@@ -60,11 +66,16 @@ Ubuntu doesn't provide a package but you can download it directly from the
...
    sudo apt install openjdk-8-jre-headless software-properties-common
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: Also, i think - per Santi's instructions, that this needs to be: 

```
sudo apt install openjdk-11-jre-headless software-properties-common
```

@scollazo is that correct?

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 6]
```diff
@@ -497,13 +508,37 @@ password ``12345`` and the permissions needed for the database created above.
...
-   mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
+   sudo mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
 
 Note that the ``INDEX``, ``CREATE`` and ``ALTER`` privileges are only necessary
 during the installation process or when you are upgrading AtoM to a newer
 version. They can be removed from the user once you are finished with the
 installation or you can change the user used by AtoM in :ref:`config.php <config-config-php>`.
 
+Finally, let’s configure our MySQL modes. The The MySQL server can operate in
+different SQL modes, affect the SQL syntax MySQL supports and the data
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: missing "which" after the comma here (sorry if that was my typo in the notes!)

File: admin-manual/installation/linux/ubuntu-xenial.rst [hunk 7]
```diff
@@ -70,12 +76,18 @@ Ubuntu doesn't provide a package but you can download it directly from the
...
 
+   code content
+
 .. code-block:: bash
 
-   sudo apt install openjdk-8-jre-headless software-properties-common
+   sudo add-apt-repository ppa:openjdk-r/ppa
+   sudo apt-get update
+
+   sudo apt install openjdk-11-jre-headless software-properties-common
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-xenial.rst]: it looks like this got reversed. bionic is 18.04; xenial is 16.04 - so this one should still be `openjdk-8`

File: admin-manual/installation/linux/ubuntu-xenial.rst [hunk 8]
```diff
@@ -515,7 +527,7 @@ password ``12345`` and the permissions needed for the database created above.
 
 .. code-block:: bash
 
-   mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
+   sudo mysql -h localhost -u root -p -e "GRANT ALL ON atom.* TO 'atom'@'localhost' IDENTIFIED BY '12345';"
```
Comment (fiver-watson) [inline admin-manual/installation/linux/ubuntu-xenial.rst:530]: I think we are missing the section on SQL modes that we added to the Bionic / 18.04 docs - we should add it here too.

**Review by fiver-watson — CHANGES_REQUESTED** [review]
I've flagged a few minor things, but this is looking v close! Thanks!

**Review by fiver-watson — APPROVED** [review]
Looks great, thanks! :D

File: admin-manual/installation/linux/ubuntu-bionic.rst (hunk 5 above)
Comment (scollazo) [reply admin-manual/installation/linux/ubuntu-bionic.rst]: I think that sofwar-properties-common is not needed in 18.04, and the apt-get update command should be the fist one

**Pull request merged**